hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieMultiTableDeltaStreamer.java

```diff
@@ -245,10 +246,13 @@ public void testTableLevelProperties() throws IOException {
         case "dummy_table_short_trip":
           String tableLevelKeyGeneratorClass = tableExecutionContext.getProperties().getString(DataSourceWriteOptions.KEYGENERATOR_CLASS_NAME().key());
           assertEquals(TestHoodieDeltaStreamer.TestTableLevelGenerator.class.getName(), tableLevelKeyGeneratorClass);
+          List<String> transformerClass = tableExecutionContext.getConfig().transformerClassNames;
```

Comment:
I've added these as tests for checking the transformers. Let me know your thoughts on this. I'm not sure if this is the what you expect.

I'm yet to look at the failed builds. And sure. Will rebase to latest master.

Reply:
@nsivabalan